src/components/UserList/index.tsx

```diff
@@ -76,9 +77,74 @@ const messages = defineMessages({
   sortRequests: 'Request Count',
   localLoginDisabled:
     'The <strong>Enable Local Sign-In</strong> setting is currently disabled.',
+  neverSignedIn: 'Never Signed In',
+  activeNow: 'Active Now',
+  lastActive: 'Last Active',
 });
 
-type Sort = 'created' | 'updated' | 'requests' | 'displayname';
+
+// Helper function to format the last active time
+const formatLastActive = (lastActive: string | null, intl: IntlShape): string => {
```

Comment:
You can use formatRelativeTime instead of doing all of this manually. Look at other examples in Overseerr on how to use it.